resources/serverless/charts/docker-registry/values.yaml

```diff
@@ -116,12 +116,49 @@ configData: # example: https://github.com/docker/distribution/blob/master/cmd/re
       interval: 10s
       threshold: 3
 
-securityContext:
-  enabled: true
-  runAsUser: 1000
-  fsGroup: 1000
-  privileged: false
-  allowPrivilegeEscalation: false
+containers:
+  securityContext:
+    runAsUser: 1000
+    fsGroup: 1000
+    privileged: false
+    allowPrivilegeEscalation: false
+    # capabilities:
+    #     add: # Only the following capabilities are allowed in the baseline standard. If no capabilities needed, the whole add-section has to be removed.
+    #       - AUDIT_WRITE
+    #       - CHOWN
+    #       - DAC_OVERRIDE
+    #       - FOWNER
+    #       - FSETID
+    #       - KILL
+    #       - MKNOD
+    #       - NET_BIND_SERVICE
+    #       - SETFCAP
+    #       - SETGID
+    #       - SETPCAP
+    #       - SETUID
+    #       - SYS_CHROOT
+    # seLinuxOptions: # Optional. Set this on container level and only if it is really necessary. Setting the user or role option is forbidden. If the setting is not needed, the whole seLinuxOptions-section has to be removed.
+    #   type: container_t # or container_init_t or container_kvm_t
+    procMount: default # Optional. The default is false if the entry is not there.
+    readOnlyRootFilesystem: true # Mandatory
+
+pod:
+  securityContext:
+    runAsUser: 1000
+    runAsGroup: 1000
```

Comment:
Based on the document located [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#podsecuritycontext-v1-core) I think more important fields are under the containers security context. Is it possible to move `runAsUser` and `runAsGroup` under the `container.securityContext`? What do you think?

Reply:
I think they are explicitly required by the admission controller here as well.

Reply:
For sure the `runAsUser` and `runAsGroup` on the container level are not needed. Here @anoipm decided to set these fields under the pod's template only. What do you think about it? Sorry, I said that more important fields are under the `containers security context` but that's not true 😅 I mean we can use the pod's template to not duplicate these fields. Security context on the container (or initContainer) level gets values from the pod's security context first or overrides them if these on the container level are defined.

Reply:
I can remove them from the container/initcontainer security context. I was trying to follow the explicit example provided by the security team.

Reply:
this explicit example is definitely too detailed :/ Here is an article about [Pod Security Standard](https://kubernetes.io/docs/concepts/security/pod-security-standards/) where we can see that most of the fields used in examples don't need to be even specified.